lib/dateConversion.js

```diff
@@ -0,0 +1,12 @@
+module.exports.convertToISODateString = (dateString) => {
+  if (dateString) {
+    try {
+      return new Date(dateString).toISOString();
+    } catch (err) {
+      console.log(`indexed-date can not be converted to ISO standard date`);
```

Comment:
This error message seems to imply that this function should only be used for validating `indexed-date`, yet the name and location of it makes it look like a more generic function.

Reply:
Yup, good catch, I started off just doing indexed, then went generic, I'll update